src/components/Technicians/TechniciansForm/index.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../sharedComponents/Button';
+import Select from '../../sharedComponents/Select';
+import styles from './TechniciansForm.module.css';
+
+const TechniciansForm = ({
+  onSubmit,
+  onClose,
+  technician,
+}) => {
+  const [state, setState] = useState({
+    firstName: technician.firstName || 'Insert first name',
```

Comment:
The default values should be empty strings
```suggestion
    firstName: technician.firstName || '',
```

If you want to add those values to the inputs, you have to use the placeholders.